Source/Fuse.Sensor/Android/data/BatteryData.java

```diff
@@ -0,0 +1,150 @@
+package com.fuse.sensorkit;
+
+import android.os.BatteryManager;
+import java.util.Locale;
+
+import static android.os.BatteryManager.*;
+
+public class BatteryData {
+
+	protected final int level;
+    protected final int scale;
```

Comment:
Looks like there's some mixed indentation here

Reply:
oops! sorry about that, fix is on the way..